lua-cjson.rockspec

```diff
@@ -0,0 +1,40 @@
+package = "lua-cjson"
```

Comment:
question: is this for manual/automatic testing?
I would put something like this in `rocks-lib/resources/test/`

Reply:
Manual testing, I'll be removing it. It got added automatically on accident because I use `jj` instead of `git` hahah